contracts/libs/TokenRouter.sol

```diff
@@ -52,16 +52,19 @@ abstract contract TokenRouter is Router {
         bytes32 _recipient,
         uint256 _amount
```

Comment:
Since this is also used for 721, `_amount` has been a confusing argument name. Would recommend to rename it to `amountOrId` or have separate base classes